docs/user/site.rst

```diff
@@ -229,6 +229,16 @@ mesh_vpn
                   -- You can also omit the ipv4 to allow both connection via ipv4 and ipv6
                   remotes = {'"vpn2.alpha-centauri.freifunk.net" port 10000'},
                 },
+                peer3 = {
+                  key = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+                  -- In addition to domains you can also add ip addresses, which provides
+                  -- resilience in case of dns outages
+                  remotes = {
+                    '"vpn3.alpha-centauri.freifunk.net" port 10000',
+                    'ipv6 "2001:DB8::3:1" port 10000',
+                    'ipv4 "192.0.2.3" port 10000',
```

Comment:
IP addresses should not be enclosed in double quotes, that will spare fastd running them through the resolver. The *ipv4*/*ipv6* keywords are redundant in this case. I'd prefer lowercase characters for hex digits in IPv6 addresses.